db/migrate/20161214140124_add_meta_to_events.rb

```diff
@@ -0,0 +1,5 @@
+class AddMetaToEvents < ActiveRecord::Migration[5.0]
+  def change
+    add_column :events, :meta, :json
```

Comment:
This new column can be used in diverse ways, depending on the event type. The only thing to be VERY careful of is consistency.